scalding-core/src/test/scala/com/twitter/scalding/TypedPipeTest.scala

```diff
@@ -44,7 +44,7 @@ class TupleAdderTest extends WordSpec with Matchers {
   import Dsl._
   "A TupleAdderJob" should {
     JobTest(new TupleAdderJob(_))
-      .source(TypedTsv[(String, String)]("input", ('a, 'b)), List(("a", "a"), ("b", "b")))
```

Comment:
let's not make this change. This will appear in a code in science and elsewhere, and if it is easy for us to just support it, we should.